actors/runtime/proof/verify.go

```diff
@@ -28,6 +29,18 @@ type AggregateSealVerifyInfo = proof5.AggregateSealVerifyInfo
 
 type AggregateSealVerifyProofAndInfos = proof5.AggregateSealVerifyProofAndInfos
 
+///
+/// Replica
+///
+
+// Information needed to verify a replica update
+
+type ReplicaUpdateInfo struct {
```

Comment:
You will need a proof too.  Maybe some other things too but I'll check up on that and get back to you.

Reply:
I suppose I can be convinced that including a proof in here might be beneficial.